src/target/tx/devo/common/Makefile.inc

```diff
@@ -82,6 +85,16 @@ ALL = $(LIBOPENCM3) $(TARGET).dfu
 
 else #BUILD_TARGET
 
+ifdef ENABLE_LTO
+CFLAGS_pwm_isr = -fno-lto
+CFLAGS_err_isr = -fno-lto
+CFLAGS_clock_isr = -fno-lto
+CFLAGS_usb_isr = -fno-lto
+CFLAGS_ppmin_isr = -fno-lto
+CFLAGS_uart_isr = -fno-lto
+CFLAGS_soft_serial_isr = -fno-lto
+endif
+
 ifeq ($(OS),Windows_NT)
```

Comment:
How about something like:
```
$(foreach src,$(filter %_isr.c,$(SRCS)),\
            $(eval CFLAG_$(src) = -fno-lto))
```

Reply:
sure. I like this solution. but not sure how i should use this in makefile.

Reply:
I think it will just work as is?  Probably need to be tweaked to look like:
```
$(foreach src,$basename $(notdir $(filter %_isr.c,$(SRCS)))),\
            $(eval CFLAG_$(src) = -fno-lto))
```
Try just using this as is instead of your current CFLAG definitions